ember-phone-input/src/unpublished-developer-types/index.d.ts

```diff
@@ -0,0 +1,11 @@
+import '@glint/environment-ember-loose';
+import 'ember-source/types';
+import 'ember-source/types/preview';
+import type RenderModifiersRegistry from '@ember/render-modifiers/template-registry';
```

Comment:
Are you sure a template registry is exposed?

Reply:
It did work on the WIP migration and is per [docs](https://github.com/emberjs/ember-render-modifiers#glint-usage) so I think it should be fine

Reply:
Nice! I missed that when doing this work for [ember-lottie](https://github.com/qonto/ember-lottie/blob/main/ember-lottie/src/unpublished-development-types/index.d.ts). Will try to update this there